Introductory Tests/traj_opt.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    x = pop.champion_x
</pre><p>maybe clear the output of this cell to avoid clogging the repo, it's a loooot of lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>